scripts/linters/custom_eslint_checks/lib/index.js

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview ESLint plugin for Oppia codebase which indexes all custom
```

Comment:
Write about the file here! I'm not sure what's `which` here? I think I had suggested a better fileoverview for this file earlier, can you please use that or something around that?